layers/+lang/python/packages.el

```diff
@@ -351,6 +351,8 @@
         "ri" 'spacemacs/python-remove-unused-imports
         "sB" 'spacemacs/python-shell-send-buffer-switch
         "sb" 'spacemacs/python-shell-send-buffer
+        "sE" 'spacemacs/python-shell-send-statement-switch
```

Comment:
This binding should also be documented.

Reply:
Hi @smile13241324 , Yes, you're right, I add it into the Readme.org now. 
Please review again.
Thank you

Reply:
https://github.com/syl20bnr/spacemacs/commit/deb13cd2faf726d275b86b9622cca4f413970b1f#diff-069ce66d5b3d471d2ab7f214fab50ae7R446